README.md

```diff
@@ -282,17 +266,6 @@ For example, here is `jest` running on all `.js` files with the `NODE_ENV` varia
 }
 ```
 
-### Use ng lint with angular cli >= 7.0.0
```

Comment:
Can we move the example to the section with functions and show how to substitute `relative`?